ruby/command-t/scanner/file_scanner/find_file_scanner.rb

```diff
@@ -36,8 +36,9 @@ def paths!
             counter = 1
             next_progress = progress_reporter.update(counter)
             stdout.each_line do |line|
-              next if path_excluded?(line.chomp!)
-              paths << line[@prefix_len..-1]
+              line = line[@prefix_len..-2]
```

Comment:
I'm pretty sure it's not (and the tests agree). Before you were removing the null, then the prefix. This just removes the null and prefix in one go. If you want I can put it back (`chomp! is pretty much free`) but I thought it looked a little tidier.